plugins/homepage/web_client/views/FrontPageView.js

```diff
@@ -10,7 +10,23 @@ wrap(FrontPageView, 'render', function (render) {
         method: 'GET',
         url: 'homepage/markdown'
     }).done(_.bind(function (resp) {
-        this.$el.html(renderMarkdown(resp['homepage.markdown']));
+        if (resp['homepage.markdown'] === '') {
+            _.bind(render, this)();
```

Comment:
I think you want `render.call(this)` here. See [these docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call) for more info.